src/main/docker/app.yml

```diff
@@ -6,6 +6,13 @@ services:
       - _JAVA_OPTIONS=-Xmx512m -Xms256m
       - SPRING_PROFILES_ACTIVE=prod,swagger
       - MANAGEMENT_METRICS_EXPORT_PROMETHEUS_ENABLED=true
+      - SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_OIDC_ISSUER_URI=http://keycloak:9080/auth/realms/jhipster
+      - SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_ID=jhipster-control-center
+      - SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_SECRET=jhipster-control-center
       - JHIPSTER_SLEEP=30 # gives time for other services to boot before the application
     ports:
       - 7419:7419
+  keycloak:
```

Comment:
Should we add keycloak in the app docker compose file as we start the image without oauth2 profile? 
Maybe we should comment those lines?